src/bundle/Resources/views/layout.html.twig

```diff
@@ -34,6 +34,13 @@
         </script>
         <script src="/bundles/fosjsrouting/js/router.js"></script>
         <script src="{{ path('fos_js_routing_js', { callback: 'fos.Router.setData' }) }}"></script>
+        {% if app.request.locale == 'ach_UG' %}
+            <script type="text/javascript">
```

Comment:
This could/should be moved to the bottom of the file. It could also be added by an external file, as it is supported by the template. Pointers appreciated.